### PR TITLE
Implement MinimalLib get_mcs() version that returns JSON

### PR DESCRIPTION
**Pull request by ptosco**
This PR implements into MinimalLib a version of `get_mcs()` that returns a JSON string rather than a SMARTS string or a `JSMol`.
This is useful to:
* tell whether an MCS calculation has run to completion or rather has timed out (by checking `canceled`)
* to retrieve the size of the MCS (by accessing `numAtoms` and `numBonds`)
* to retrieve all degenerate SMARTS as an array rather than as a string in case `StoreAll` was set to `true`

I also realized that I had forgotten to expose `StoreAll` to `parseMCSParametersJSON`, so I fixed that too.

### CODE REVIEW

File: Code/MinimalLib/minilib.cpp [hunk 1]
```diff
@@ -835,6 +835,39 @@ MCSResult getMcsResult(const JSMolList &molList,
 }
 } // namespace
 
+std::string get_mcs(const JSMolList &molList, const std::string &details_json) {
```
Comment (greglandrum) [inline Code/MinimalLib/minilib.cpp:838]: I think it would be better to be consistent here and call this `get_mcs_as_json()`

**Review by greglandrum — CHANGES_REQUESTED** [review]
Only the one suggestion about renaming the new function to make it consistent with the other mcs variants

File: Code/MinimalLib/minilib.cpp (hunk 1 above)
Comment (ptosco) [reply Code/MinimalLib/minilib.cpp]: @greglandrum Done.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**